core/templates/dev/head/pages/exploration-editor-page/editor-tab/graph-directives/state-graph-visualization.directive.ts

```diff
@@ -338,7 +338,8 @@ oppia.directive('stateGraphVisualization', [
                       // We need a separate layer here so that the translation
                       // does not influence the panning event receivers.
                       $scope.innerTransformStr = (
-                        'translate(' + (<d3.ZoomEvent>d3.event).translate + ')'
+                        'translate(' + (<d3.ZoomEvent>d3.event).transform.x +
```

Comment:
@kevintab95, do we have to change the first translate string to transform as well? (not sure!)
```suggestion
                        'transform(' + (<d3.ZoomEvent>d3.event).transform.x +
```

Reply:
I think this will be used as a CSS property, so we need to have translate here!

Reply:
Hi, i too think it should translate as it is translating a CSS property to read more about the same please visit [here](https://github.com/d3/d3-zoom/blob/master/README.md#zoom-transforms) and if this is not the case please let us know.

Reply:
Also, please check if this issue [#1105](https://github.com/oppia/oppia/issues/1105) is still reproducible if not you can close the issue.